### PR TITLE
docs: power states revamp

### PR DESCRIPTION
**Pull request by Kimberly-Davis**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

        modified:   docs/Reference/powerstates.md
        modified:   docs/Reference/troubleshooting.md
        modified:   mkdocs.yml
## Anything the reviewer should know when reviewing this PR?
Reorganized power states section.
NOTE: A very first pass on helping out DNS pathfinding. Like, minimal. Working on DNS explication in second PR.

### CODE REVIEW

File: docs/Reference/powerstates.md [hunk 1]
```diff
@@ -2,32 +2,43 @@
...
+   | **10** | Reset | Powered up/on | Perform hardware reset on the bus | N/A | No |
+   | **12** | Power down/off (soft) | Powered up/on | Transition to a very minimal power state | G2/S5 | Yes |
+   | **14** | Soft reset | Powered up/on | Perform a shutdown and then a hardware reset | N/A | Yes |
+   | **100** | Power up to BIOS settings | Powered down/off | Power to BIOS to verify or modify system configuration | G2/S5 | Yes  |
+   | **101** | Reset to BIOS settings | Powered up/on | Perform hardware reset on the bus to BIOS to verify or modify system configuration | G2/S5 | Yes  |
+   | **104** | Reset to secure erase | Powered up/on |  Perform hardware reset on the bus to secure erase, a process of completely erasing a solid state drive (SSD) | G2/S5 | Yes |
+
+## Alternative Boot Options
+
+Currently, the Toolkit doesn't natively support the 200 and 400-level calls.
```
Comment (matt-primrose) [inline docs/Reference/powerstates.md:23]: I thought the 400-level calls did work?  @bwendlandt-intel  can you confirm if these work?  I know the 200-level calls are not supported.

Comment (Kimberly-Davis) [reply docs/Reference/powerstates.md:23]: So, it's just 200 that aren't supported then?

Comment (bwendlandt-intel) [reply docs/Reference/powerstates.md:23]: @matt-primrose Do we support 104? Isn't RSE still in backlog, or is this different?

Comment (Kimberly-Davis) [reply docs/Reference/powerstates.md]: @matt-primrose @bwendlandt-intel Let me know if I need to remove 104. I did place the 400s back in the first table.

Comment (Kimberly-Davis) [reply docs/Reference/powerstates.md]: Done.

**Review by bwendlandt-intel — APPROVED** [review]

**Pull request merged**